From: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
Date: Fri, 20 May 2016 11:07:09 -0500
Subject: apple: set out_icon_name for WWAN connections for indicator modes

Otherwise one would see WWAN use the last icon applied in the applet
since indicators cannot use pixmaps for the main indicator icon.

(cherry picked from commit 73b9a58286bef20741fb66f3bebe3399b3da29e6)
---
 src/mobile-helpers.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/mobile-helpers.c b/src/mobile-helpers.c
index 751bea1..76294a1 100644
--- a/src/mobile-helpers.c
+++ b/src/mobile-helpers.c
@@ -616,6 +616,9 @@ mobile_helper_get_icon (NMDevice *device,
 		                                               mb_state,
 		                                               mb_tech,
 		                                               applet);
+		*out_icon_name = mobile_helper_get_quality_icon_name (quality_valid ?
+		                                                      quality : 0);
+
 		if ((mb_state != MB_STATE_UNKNOWN) && quality_valid) {
 			gboolean roaming = (mb_state == MB_STATE_ROAMING);
 
